src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -5,12 +5,37 @@ var LeafletPlainLayerView = require('./leaflet-plain-layer-view');
 var LeafletCartoDBLayerGroupView = require('./leaflet-cartodb-layer-group-view');
 var LeafletTorqueLayerView = require('./leaflet-torque-layer-view');
 var LeafletCartoDBWebglLayerGroupView = require('./leaflet-cartodb-webgl-layer-group-view');
+// Should we expose a Tangram.cartodb method for doing this?
+var CCSS = require('tangram-cartocss');
 
-var LayerGroupViewConstructor = function (layerGroupModel, mapModel, options) {
-  if (options.vector) {
-    return new LeafletCartoDBWebglLayerGroupView(layerGroupModel, mapModel);
+// Make this configurable and crate a pipe of decision based on the configuration.
+var shouldLoadWithVector = function (metadata, cb) {
```

Comment:
IMO, we should try to not "leak" the response from the Maps API into the rest of the app. Instead, I think we should set the relevant info into one of our models (`model-updater` should take care of this). Otherwise we're coupling this code to the response from Maps API. Do you agree @donflopez?